src/main/java/edu/indiana/dlib/amppd/controller/BatchController.java

```diff
@@ -100,4 +106,23 @@ public String runPreprocessingOnFilesWithoutMediaInfo(@RequestParam(value = "pri
 
 		return String.format("Files initially missing media info: %s\nFiles with media info successfully added: %s\nFiles still missing media info: %s", missingMediaInfo, success, (missingMediaInfo - success));
 	}
+	
+	/**
+	 * Serve the  fileContent  from resources/static Folder based on given fileName.
+	 * @param fileName 
+	 * @return the  content of the  file
+	 */
+	  @GetMapping("/download/{fileName:.+}")
+	    public ResponseEntity<Resource> serveFile(@PathVariable String fileName) throws Exception{
```

Comment:
Because this is accepting a generic request for a file, we should probably return a 404 if the resource isn't found on file system.  I assume this would throw a null pointer exception (500) or something similar if the file isn't found.